msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -36,22 +33,24 @@ You can perform the following steps to add SSO using Teams Toolkit in Visual Stu
 
 1. Open **Visual Studio Code**.
 2. Select **Teams Toolkit** from the Visual Studio Code activity bar.
-3. Select **Add features** under **DEVELOPMENT**.
-
-   :::image type="content" source="../assets/images/teams-toolkit-v2/add-sso/sso-add features_1.png" alt-text="Screenshot shows the Add features option under the Development option in the Visual Studio Code.":::
+3. Select **View How-to Guides** in the **DEVELOPMENT** section.
 
-   * You can select **View** > **Command Palette...** to view **Add features** window.
+   :::image type="content" source="../assets/images/teams-toolkit-v2/manual/select-feature123_1.png" alt-text="Screenshot showing the selection of View How to guides. ":::
 
-4. Select **Single Sign-On**.
+4. From the dropdown list, Select **Develop Single Sign-On Experience in Teams**. You will be redirected to the respective How-to Guide.
 
    :::image type="content" source="../assets/images/teams-toolkit-v2/add-sso/sso-select features_1.png" alt-text="Screenshot shows the Single Sign-on feature highlighted in red in the Visual Studio Code.":::
 
+   |**Development** | **How to guide** |
+   | -------- | --------|
+   |Develop Single Sign-on Experience in Teams | [How to Develop Single Sign-on Experience](https://github.com/OfficeDev/TeamsFx/wiki/Develop-single-sign-on-experience-in-Teams) |
+
 ### Add SSO using TeamsFx CLI
```

Comment:
We don't have CLI command for adding sso. Please help to remove this part.

Reply:
Updated.